presto-main/src/main/java/com/facebook/presto/sql/planner/plan/AggregationNode.java

```diff
@@ -232,16 +258,22 @@ public boolean isDecomposable(FunctionRegistry functionRegistry)
         private final FunctionCall call;
         private final Signature signature;
         private final Optional<Symbol> mask;
+        private final List<Symbol> orderBy;
+        private final List<SortOrder> ordering;
 
         @JsonCreator
         public Aggregation(
                 @JsonProperty("call") FunctionCall call,
                 @JsonProperty("signature") Signature signature,
-                @JsonProperty("mask") Optional<Symbol> mask)
+                @JsonProperty("mask") Optional<Symbol> mask,
+                @JsonProperty("orderBy") List<Symbol> orderBy,
+                @JsonProperty("ordering") List<SortOrder> ordering)
         {
             this.call = call;
             this.signature = signature;
             this.mask = mask;
+            this.orderBy = orderBy;
+            this.ordering = ordering;
```

Comment:
* It's not validated because others are not validated. I figured I'd keep them looking the same. Should I validate them all?
* Will add
* Yes there's no existing structure wrapping them. I wish they were together. Can probably add a refactor diff in if you want. Or we can leave this for later since we do have a "end of H1" coming up here.
* You mean changing it to use empty list to mean no order by rather than using Optional.empty()? There are quite some Optional stuff (not necessarily list so it might not have the luxury to use empty list to mean none) so I kept Optional.